docs/source/general/contributing.rst

```diff
@@ -1,14 +1,182 @@
 Contributing
 ==================== 
 
-Contributing 
------------------
+.. contents:: Overview
+   :depth: 2
+   :local: 
 
-Using Coverage (Not currently implement in CI) :
+Getting Started
+----------------- 
+We are happy your are even considering helping. We hope this guide makes the process seamless. We really appreciate your help! 
 
-`Coverage <https://coverage.readthedocs.io/en/latest/>`_ is a tool used to monitor test coverage. It does so by excuting the tests and monitoring which lines are run. 
+If you are new to open source contribution, please checkout :ref:`best-practices` and :ref:`toolbelt`. 
 
-To use it from your command line: 
+.. note:: 
+   All command line code is run from the project root except where noted.
+
+.. tip:: 
+   In :code:`git clone <forked_repo_https_link.git>` , text inside the < >, and the < > get replaced with the appropriate specific text. The < > visually separate the syntax which remains unchanged from the text that needs to be changed. It is a common development pattern that is confusing if you have not seen it before. To see another example of it, type :code:`git --help` on your command line.
+
+Select an issue 
+----------------
+Look through the open issues and for an issue labelled ``good first issue`` or ``documentation`` that is unassigned. 
+
+Once you have selected an issue, then assign it to yourself by placing the word ``take`` as a comment. This will initiate a github action to assign it to you. It may take a few seconds or require you to refresh the page. Your goal is to finish the process from self assigning the issue to submitting the initial Pull Request (PR) in less than 2 weeks. This keeps your work from diverging too much from the main branch. 
```

Comment:
```suggestion
Once you have selected an issue, then assign it to yourself by placing the word ``take`` as a comment. This will indicate to other contributors that you're working on the issue. Your goal is to finish the process from self assigning the issue to submitting the initial Pull Request (PR) in less than 2 weeks. This keeps your work from diverging too much from the main branch. 
```

Pulling out the action specific language